### PR TITLE
Stats: Adding empty state component for Clicks

### PR DESCRIPTION
**Pull request by annacmc**
Related to [#64](https://github.com/Automattic/red-team/issues/64)

## Proposed Changes

* adds a new empty state for Clicks

## Testing Instructions

* go to the live branch
* apply `stats/empty-module-traffic` feature flag
* verify that a page with empty Clicks views component works with and without the feature flag
* repeat for blogs with data

image

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

File: client/my-sites/stats/site.jsx [hunk 1]
```diff
@@ -498,27 +499,37 @@ class StatsSite extends Component {
...
+						{ isNewStateEnabled && (
+							<StatsModuleClicks
+								path="clicks"
+								moduleStrings={ moduleStrings.clicks }
+								period={ this.props.period }
+								query={ query }
+								statType="statsClicks"
+								showSummaryLink
+								className={ clsx(
+									'stats__flexible-grid-item--40--once-space',
```
Comment (grzegorz-cp) [inline client/my-sites/stats/site.jsx]: We need to maintain the same classes to allow for proper grid rendering across environments.

**Review by grzegorz-cp — CHANGES_REQUESTED** [review]
Let's maintain the CSS classes.

File: client/my-sites/stats/site.jsx (hunk 1 above)
Comment (annacmc) [reply client/my-sites/stats/site.jsx]: Oh yeah, totally - this was an oversight and I still had the commit sitting locally for this. Thanks for pointing it out :)

**Review by grzegorz-cp — APPROVED** [review]
👍

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/15839334

Some locales (Hebrew, Japanese) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Thank you @annacmc for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/15839334).

**Pull request merged**